package.json

```diff
@@ -1,6 +1,14 @@
 {
   "dependencies": {
+    "clsx": "^1.2.1",
     "express": "^4.18.2",
-    "mysql": "^2.18.1"
+    "mysql": "^2.18.1",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
```

Comment:
The only things that should be here are "express" and "mysql". The react and clsx stuff is front-end only and should be in the ./academic-requirements/package.json file instead. I don't think having them here will necessarily cause problems, but they are unnecessary dependencies for the API. 

I think because we are ignoring the node_modules folders, once you install a package it is available in all branches even if the packages are not in the branch's package.json files.

If someone doesn't have the node modules already installed, running `npm i` in the root directory and the academic-requirements directory would install any dependencies (rather than installing manually). 

In this screenshot, the package.json in the root folder should look like the one on the left of the image, the package.json in ./academic-requirements should look something like the one on the right (but with the new clsx dependency)
![image](https://user-images.githubusercontent.com/15573759/198499877-8b0dbaee-b69c-44b7-91f7-8e2aad379c87.png)